jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -129,6 +129,22 @@ private static void assertDockerInspect(String imageReference)
                 + "            }"));
   }
 
+  private static void assertExtraDirectoryDeprecationWarning(String pomXml)
+      throws DigestException, IOException, InterruptedException {
+    String targetImage = "localhost:6000/simpleimage:gradle" + System.nanoTime();
+    BuildResult buildResult =
+        JibRunHelper.buildToDockerDaemon(simpleTestProject, targetImage, pomXml);
+    Assert.assertEquals(
+        "Hello, world. \nrw-r--r--\nrw-r--r--\nfoo\ncat\n",
+        new Command("docker", "run", "--rm", targetImage).run());
+    Assert.assertThat(
+        buildResult.getOutput(),
+        CoreMatchers.containsString(
+            "'jib.extraDirectory', 'jib.extraDirectory.path', and 'jib.extraDirectory.permissions' "
+                + "are deprecated; use 'jib.extraDirectories', 'jib.extraDirectories.paths' and "
```

Comment:
If we aren't supporting `jib.extraDirectories` by itself, maybe we should remove that from this message (or rephrase to be clear about the configuration block `jib.extraDirectories { ... }`?).

Reply:
Ah, good catch!